CHANGELOG.md

```diff
@@ -2,6 +2,26 @@
 
 A breaking change will get clearly marked in this log.
 
+## Unreleased
+
+## Add
+
+- Add support for `/paths/strict-send`. ([444](https://github.com/stellar/js-stellar-sdk/pull/444))
+  This functions takes the asset you want to send, the amount and a list of
```

Comment:
This function takes the asset you want to send, and the amount of that asset, along with either a list of destination assets or a destination address.